### ISSUE TITLE
Can't translate custom tasks

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#347]
Let's include a way for users to translate custom tasks

**Comment by ojopaul** [comment]
@stevejburge, are you referring to custom texts entered via the box below? I'm not sure translation works for dynamic texts or you're referring to something else.

![custom-task](https://user-images.githubusercontent.com/92122385/162742092-2e7f5ca6-8e51-4d3e-92d4-bcb4b09a1f5d.jpg)

**Comment by stevejburge** [comment]
@ojopaul Good point. Let's skip this one for now. We'd need to think of a good way to make this multi-lingual
